src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,18 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder cleanText = new StringBuilder("");
+
+        cleanText.append(text.toUpperCase().replaceAll(" ", "")
+                .replaceAll("!", "")
```

Comment:
good start, but let's simplify this solution:
read smth about regex with replaceAll()
Some examples:
https://www.javamex.com/tutorials/regular_expressions/search_replace.shtml
https://stackoverflow.com/questions/632204/java-string-replace-using-regular-expressions